src/main/java/wolox/bootstrap/configuration/AppConfig.java

```diff
@@ -2,11 +2,9 @@
 
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
-import org.springframework.context.annotation.PropertySource;
 import wolox.bootstrap.services.EnvironmentConfigurationService;
 
 @Configuration
-@PropertySource("classpath:application.properties")
```

Comment:
This works? 😍

Reply:
Yes! The problem was the Spring Boot version 😃